scripts/config/lxdialog/checklist.c

```diff
@@ -119,7 +119,7 @@ int dialog_checklist(const char *title, const char *prompt, int height,
 	}
 
 do_resize:
-	if (getmaxy(stdscr) < (height + CHECKLIST_HEIGTH_MIN))
+	if (getmaxy(stdscr) < (height + CHECKLIST_HEIGHT_MIN))
```

Comment:
It's possible that `lxdialog` is a third party module included as a whole. If so, let me know and I'll happily add it to `excludes.txt`.